include/swift/AST/DiagnosticsSema.def

```diff
@@ -3207,6 +3207,16 @@ ERROR(reasync_without_async_parameter,none,
 ERROR(inherits_executor_without_async,none,
       "non-async functions cannot inherit an executor", ())
 
+ERROR(eagermove_requires_lexical_lifetimes, none,
+      "@_eagerMove requires lexical lifetimes to be enabled", ())
+
+ERROR(lexical_requires_lexical_lifetimes, none,
+      "@_lexical requires lexical lifetimes to be enabled", ())
```

Comment:
Disabling lexical lifetimes in the compiler should just override attributes rather than breaking source. No need for a diagnostic.

Reply:
Removed the diagnostic.